styleguide/CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+## Working with Pattern Lab
+1. All work is done in the `/styleguide/source` directory.
+    - We use [BEM](http://getbem.com/introduction/) to structure our scss which gets compiled to css during build and automatically with the gulp task running.
+    - Most patterns will have a `.twig` [markup file](https://twig.sensiolabs.org/), `.json` default [data file](http://patternlab.io/docs/data-pattern-specific.html), and `.md` [documentation file](http://patternlab.io/docs/pattern-documenting.html) in the `/styleguide/source/_patterns` directory.
+    - If a pattern has styles associated with it, the corresponding `.scss` files can be found in the corresponding folder.
+     - If a pattern has js functionality associated with it, the corresponding `.js` file can be found in the `/styleguide/source/assets/js/` directory.
```

Comment:
As a rule we wanted to have JS be stored in the pattern directory. We made a push to move JS from the `assets` dir a while back, but there still some leftover stuff there. Would you mind updating the language here with a note about that? (Or if you think we should do it differently, open to hearing about that too.)